main/src/ca/uwaterloo/flix/language/phase/jvm/AsmOps.scala

```diff
@@ -541,7 +541,7 @@ object AsmOps {
     mv.visitInsn(ACONST_NULL)
 
     // Construct the proxy object.
```

Comment:
Is this method even used?

Reply:
Its used in relation to function application but right now it uses `flix/runtime/ProxyObject` while the jvm name refers to `JvmName(List("dev", "flix", "runtime"), "ProxyObject")`. It seems like it isn't used somehow but it's definitely called.. Im not sure

Reply:
I've fixed the reference now. I'll look further into it when it gets its turn during the dsl iteration